tests/runtimes/simple-keyvalue/Cargo.toml

```diff
@@ -17,7 +17,7 @@ path = "src/upgraded.rs"
 [package.metadata.fortanix-sgx]
 heap-size = 134217728
 stack-size = 2097152
```

Comment:
I assume stack and heap size values are large enough to handle the potentially increased memory usage due to the more threads?

Reply:
Heap size likely needs to be bumped as query caches are thread-local. Will do some adjustments.

Reply:
Stack size is per-thread so this should be fine. Bumped heap size and reduced the number of worker threads for now as we can also do internal load balancing across multiple runtime enclaves if needed.